cypress/integration/interface_builder_test.js

```diff
@@ -29,15 +29,20 @@ describe('Interface builder tests', () => {
       cy.get('#interfaceMinor').should('have.value', '1');
     });
 
-    it('shows selected interface', function() {
+    it('shows selected interface', function () {
       const testInterfaceName = this.test_interface.data.interface_name;
       const testInterfaceMajor = this.test_interface.data.version_major;
       const testInterfaceMinor = this.test_interface.data.version_minor;
       cy.visit(`/interfaces/${testInterfaceName}/${testInterfaceMajor}`);
-      cy.location('pathname').should('eq', `/interfaces/${testInterfaceName}/${testInterfaceMajor}`);
+      cy.location('pathname').should(
```

Comment:
i see some reformatting diffs. Should we check for format in the cypress tests too?
If so, we should add CI checks to prevent PRs with wrong formatting

Reply:
My bad, the IDE has a format-on-save that follows the rules we specified for the react codebase.
I think we can add a PR to extend the files formatted to include Cypress tests.
For now I'll just take out the reformatting from this commit